index.html

```diff
@@ -1,13 +1,149 @@
+          <li><a href="#">Sponsor</a></li>
+          <li><a href="#">News</a></li>
+        </ul>
+      </div>
+    </nav>
+    <section id="header">
+      <div>
+        <blockquote>"Designing the future"</blockquote>
+        <h1 class="header">International <br> <span>Technologies</span> <br> Conference</h1>
+      </div>
+      <div class="introduction">
+        <p><span class="quote">"The most important technologies conference in the world"</span> <span class="author">Technologies Magazine</span></p>
+        <p>The IT Conference will be held in Paris. The most successful technology leaders will present on the future of technology.</p>
+      </div>
+      <div class="place">
+        <p class="date">2022.10.12 ~ 17 (saturday)</p>
+        <p class="salon">@ Le salon de Paris, Paris, France</p>
+      </div>
+    </section>
+
+    <main>
+      <!-- Program Section -->
+      <section id="program">
+        <!-- cross bg -->
+        <div class="cross-bg">
+          <h2 class="program-heading">Main Program</h2>
+          <div class="line"></div>
+
+          <!-- programs-container -->
+          <div class="programs-container">
+            <div class="program-card">
+              <img src="./src/img/icons/program_icon_01.png" alt="lecture icon">
+              <h3 class="program-card-heading">Lecture</h3>
+              <p class="program-desc">Listen to speakers from various tech companies about the messages of tech future.</p>
+            </div>
+            <div class="program-card">
+              <img src="./src/img/icons/program_icon_02.png" alt="exhibition icon">
+              <h3 class="program-card-heading">Lecture</h3>
+              <p class="program-desc">Let's get to know the creations of artists from various fields who share the open spirit of ITC.</p>
+            </div>
+            <div class="program-card">
+              <img src="./src/img/icons/program_icon_03.png" alt="forum icon">
+              <h3 class="program-card-heading">Lecture</h3>
+              <p class="program-desc">We have time to share our thoughts and opinions with experts by topic.</p>
+            </div>
+            <div class="program-card">
+              <img src="./src/img/icons/program_icon_04.png" alt="workshop icon">
+              <h3 class="program-card-heading">Lecture</h3>
+              <p class="program-desc">You can create your own creations using open source code instead of just looking at them.</p>
+            </div>
+            <div class="program-card">
+              <img src="./src/img/icons/program_icon_05.png" alt="ignite icon">
+              <h3 class="program-card-heading">Lecture</h3>
+              <p class="program-desc">Create free networking opportunities with ITC staff from around the world.</p>
+            </div>
+          </div>
+          <!-- programs-container -->
+          <button class="program-button"><a class="program-link" href="#">See the whole program</a></button>
```

Comment:
- [ ] please add the `see the whole program` link on the desktop version
- [ ] kindly use the same color and design for the `see the whole program` button on the mobile version

your version
![Screenshot (455)_LI](https://user-images.githubusercontent.com/65298048/190790932-b2ee3f63-3a7d-4ffc-b388-5f53500c0986.jpg)
original version
![Screenshot (454)](https://user-images.githubusercontent.com/65298048/190791423-6323f515-d8d4-4375-bfb2-12b9e23f6ce6.png)